package/CHANGELOG

```diff
@@ -86,6 +86,7 @@ Fixes
   * Contact Analysis class respects PBC (Issue #2368)
 
 Enhancements
+  * Added computation of Mean Squared Displacements (#2438, PR #2619)
```

Comment:
Also add to Changes that tidynamics is now required.

Also add it to https://github.com/MDAnalysis/mdanalysis/wiki/Style-Guide#list-of-core-module-dependencies (not sure if this list is duplicated in the User Guide – @lilyminium ?)

Please raise an issue in https://github.com/conda-forge/mdanalysis-feedstock that tidynamics be added.

Reply:
It is -- I have a [somewhat related issue going](https://github.com/MDAnalysis/UserGuide/issues/66). However, I thought core dependencies did not include libraries exclusive to analysis modules?

Reply:
Oops – you're right. `tidynamics` isn't core. Sorry – ignore this one.

Reply:
More precisely: ignore anything that means "tidynamics is a core dependency". It still needs to be added to the feedstock so that it gets installed with `conda install mdanalysis`.

How does your code handle the absence of `tidynamics`?

Reply:
can I resolve this? :)